docs/src/folderstruct.md

```diff
@@ -0,0 +1,55 @@
+# ClimaLSM folder structure
+
+This documentation page is meant to help new users navigate through the repository of [ClimaLSM](https://github.com/CliMA/ClimaLSM.jl).
+
+`ClimaLSM` home directory has 5 main folders:<br>
```

Comment:
can remove the <br> (it appears for me in the rendered page?)

Reply:
I think you mean to remove the \<br\> (it got removed from the markdown comment). It doesn't look like it shows up in the rendered page for me though (looking at "Preview")